A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_CRZ.js

```diff
@@ -67,67 +86,102 @@ var A320_Neo_LowerECAM_CRZ;
             }
             
             // Fuel
-            var unitFactor = SimVar.GetSimVarValue("FUEL WEIGHT PER GALLON", "lbs");
-            if (BaseAirliners.unitIsMetric(Aircraft.A320_NEO)) {
-                unitFactor = SimVar.GetSimVarValue("FUEL WEIGHT PER GALLON", "kilogram");
-            }
-            let leftConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:1", "gallon") * unitFactor * 0.001;
-            let rightConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:2", "gallon") * unitFactor * 0.001;
+            const leftConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:1", "gallon") * this.unitFactor * 0.001;
+            const rightConsumption = SimVar.GetSimVarValue("GENERAL ENG FUEL USED SINCE START:2", "gallon") * this.unitFactor * 0.001;
+
+            const leftConsumptionShown  = fastToFixed(leftConsumption - (leftConsumption % 10), 0);
+            const rightConsumptionShown = fastToFixed(rightConsumption - (rightConsumption % 10), 0);
 
-            this.FuelUsedTotal.textContent = Math.round(leftConsumption)+Math.round(rightConsumption);
-            this.FuelUsedLeft.textContent = Math.round(leftConsumption);
-            this.FuelUsedRight.textContent = Math.round(rightConsumption);
+            if ((leftConsumptionShown != this.LeftConsumptionDisplayed) || (rightConsumptionShown != this.RightConsumptionDisplayed)) {
+                const totalConsumptionShown = parseInt(leftConsumptionShown) + parseInt(rightConsumptionShown);
+                this.FuelUsedTotal.textContent = totalConsumptionShown;
+                this.FuelUsedLeft.textContent = leftConsumptionShown;
+                this.FuelUsedRight.textContent = rightConsumptionShown;
+
+                this.LeftConsumptionDisplayed  = leftConsumptionShown;
+                this.RightConsumptionDisplayed = rightConsumptionShown;
+            }
 
             // Oil
             let value = SimVar.GetSimVarValue("ENG OIL QUANTITY:1", "percent") * 0.01 * 25;
-            this.OilQuantityLeft.textContent = value.toFixed(1).split(".", 2)[0] + ".";
-            this.OilQuantityLeftDecimal.textContent = value.toFixed(1).split(".", 2)[1];
+            let valueShown = fastToFixed(value, 1);
+            if (valueShown != this.OilQuantityLeftDisplayed) {
+                const decimalSplit = value.toFixed(1).split(".", 2);
```

Comment:
fun fact for the future. You can use destructuring to give array values names.

```
const [firstValue, secondValue] = value.toFixed(1).split(".", 2);
```

Reply:
Also maybe `fastToFixed()` should be used instead? I'm not sure how much faster it is but couldn't hurt.

Reply:
I tried to use it where possible, but it has a huge flaw: When rounding to .0, it cuts the decimal. Really stupid behavior if you ask me.
I figured baking in checks to catch that is worse than just using toFixed. But now I have a weird mix of both functions and I don't really like it. I'll think about it again.

Reply:
> fun fact for the future. You can use destructuring to give array values names.
> 
> 
> 
> ```
> 
> const [firstValue, secondValue] = value.toFixed(1).split(".", 2);
> 
> ```

Uuhh thanks, I didn't know!

Reply:
yeaaaa I just noticed that bit about fastToFixed a couple hours ago... thats no good 🙃
WELP oh well.